erpnext/translations/fr.csv

```diff
@@ -271,7 +271,7 @@ Assessment Report,Rapport d'Évaluation,
 Assessment Reports,Rapports d'évaluation,
 Assessment Result,Résultat de l'Évaluation,
 Assessment Result record {0} already exists.,Le Résultat d'Évaluation {0} existe déjà.,
-Asset,Atout,
+Asset,Actif - Immo.,
```

Comment:
This doesn't quite feel correct. Can you share some sources? 
<img width="753" alt="Screenshot 2022-04-14 at 5 24 36 PM" src="https://user-images.githubusercontent.com/9079960/163385568-e10637c8-af8f-44ec-a0bf-5bd2a1db6e7b.png">

Reply:
@ankush https://github.com/frappe/erpnext/pull/30593#discussion_r844291392
Similar thing in the older PR. Seems like **Actif Immobilisation** is used as "Fixed Assets"